extensions/wikidata/tests/src/org/openrefine/wikidata/qa/scrutinizers/QuantityScrutinizerTest.java

```diff
@@ -36,67 +61,190 @@ public EditScrutinizer getScrutinizer() {
     
     @Test
     public void testBoundsAllowed() {
-        scrutinize(valueWithBounds);
+        ItemIdValue idA = TestingData.existingId;
+        Snak mainSnak = Datamodel.makeSomeValueSnak(propertyIdValue);
+        Statement statement = new StatementImpl("P1083", mainSnak, idA);
+        ItemUpdate update = new ItemUpdateBuilder(idA).addStatement(statement).build();
+
+        ConstraintFetcher fetcher = mock(ConstraintFetcher.class);
+        when(fetcher.getConstraintsByType(propertyIdValue, "Q51723761")).thenReturn(new ArrayList<>());
```

Comment:
It would be better to reuse the constant declared in the scrutinizer, NO_BOUNDS_CONSTRAINT_QID (and same for all other tests).